src/test/kotlin/uk/gov/dluhc/printapi/testsupport/testdata/entity/PrintDetailsBuilder.kt

```diff
@@ -69,7 +72,17 @@ fun buildPrintDetails(
     eroEnglish = eroEnglish,
     eroWelsh = eroWelsh,
     batchId = batchId,
-    status = status,
+    printRequestStatuses = printRequestStatuses,
+)
+
+fun aValidPrintDetailsWithSingleStatus(
```

Comment:
I'm not clear why you are being so descriptive about fact it was a single status... Can we overload the `aValidPrintDetails` with one taking a single status and other a list?

Reply:
Thats a fair shout, I can overload the original 👍 

I guess I was coming from what we used to do in previous java projects, where we would have several builders all describing the thing to be built, eg:
```
class PizzaTestDataBuilder() {

    public static Pizza aPizza(String... toppings) ....

    public static Pizza aMeatFeastPizza() {
        return aPizza("Ham", "Chicken", "Pepperoni", "Spicy Ground Beef");
    }

    public static Pizza aPepperoniPizza() {
        return aPizza("Pepperoni");
    }
}
```

(Friday night is pizza night in the Russell household! 😁 )

Reply:
Renamed 👍